scene/resources/material.h

```diff
@@ -275,6 +275,13 @@ class BaseMaterial3D : public Material {
 		SPECULAR_MAX
 	};
 
+	enum SpecularOcclusionMode {
+		SPECULAR_OCCLUSION,
```

Comment:
```suggestion
		SPECULAR_OCCLUSION_DEFAULT,
```
I'd name this more descriptively (also this would break in C# I think, where prefix is stripped)